api/v1beta1/glance_webhook.go

```diff
@@ -142,6 +205,11 @@ func (r *Glance) ValidateUpdate(old runtime.Object) error {
 		if glanceAPI.Type != o.Spec.GlanceAPIs[key].Type {
 			return errors.New("GlanceAPI deployment layout can't be updated")
 		}
+		// Fail if an invalid configuration/layout is detected for the current
+		// glanceAPI instance
+		if r.isInvalidBackend(glanceAPI, topLevelFileBackend) {
```

Comment:
@abays even during an Update of the existing CR, we want to make sure that we don't allow to have File + Split when replicas > 0, so I reproposed the same check to this function (and moved `isInvalidBackend` into its own func).